docs/development/getting-started.rst

```diff
@@ -0,0 +1,82 @@
+Getting Started
+=================
+
+.. Note:: This is a development guide for the `j5` API.
+    If you are looking to develop a new robot API, you
+    probably want to look at `Quick Start`_.
+
+Development Process
+-------------------
+
+`j5` is developed on GitHub_ and pull requests should be submitted there. If you have write access to the repository,
+you optionally can develop your changes on a branch within the main repository. Alternatively, please fork the `j5`
+repository and pull request from there.
+
+If you are working on something that has an existing issue open on the `j5` repository, please ensure that you assign
+the issue to yourself such that duplication of work does not accidentally occur.
+
+If you need help with Git, there are plenty of good tutorials online.
```

Comment:
I think it would be more approachable for newcomers if we recommended one or two tutorials here.

I've opened #228 to track this.